web/src/containers/AssurancesAndCompliance.js

```diff
@@ -0,0 +1,125 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { t } from '../i18n';
+import { Input } from '../components/Inputs';
+import { Section, Subsection } from '../components/Section';
+import { updateApd as updateApdAction } from '../actions/apd';
+
+import regLinks from '../data/assurancesAndCompliance.yaml';
```

Comment:
my rule of thumb, documented nowhere of course :), is to alphabetize these by path level and then directory/filename... since these are all on the same level (`../`), it would be:

```
import { updateApd as updateApdAction } from '../actions/apd';
import { Input } from '../components/Inputs';
import { Section, Subsection } from '../components/Section';
import regLinks from '../data/assurancesAndCompliance.yaml';
import { t } from '../i18n';
```

if there were files in the same directory as current file, those would go above, and if there were some in `../../foo/bar` those would go below. whatcha think? feel free to ignore

Reply:
I like it!  Will reorder them.